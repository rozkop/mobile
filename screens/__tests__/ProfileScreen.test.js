import React from "react";
import renderer from "react-test-renderer";
import ProfileScreen from "../ProfileScreen";

describe("profile screen", () => {
    const component = renderer.create(<ProfileScreen />).toJSON();

    it('renders correctly', () => {
        expect(component).toMatchSnapshot();
    })
});