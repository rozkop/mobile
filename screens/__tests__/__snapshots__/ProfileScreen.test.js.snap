// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home screen renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#ffffff",
      "flex": 1,
      "justifyContent": "center",
      "padding": 24,
    }
  }
>
  <Text
    style={
      {
        "fontSize": 16,
        "marginBottom": 12,
      }
    }
  />
  <ActivityIndicator
    color="#ff6542"
    size="large"
  />
</View>
`;

exports[`profile screen renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#ffffff",
      "flex": 1,
      "justifyContent": "center",
      "padding": 24,
    }
  }
>
  <Text
    style={
      {
        "fontSize": 16,
        "marginBottom": 12,
      }
    }
  />
  <ActivityIndicator
    color="#ff6542"
    size="large"
  />
</View>
`;
